ndmg/stats/qa_skullstrip.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python
+
+"""
+ndmg.stats.qa_skullstrip
+~~~~~~~~~~~~~~~~~~~~~~
+The top level ndmg quality assurance for skull strip module.
+In this module, ndmg:
+1. Input original t1w file and the skull-striped brain file
```

Comment:
replace with `takes an original t1w nifti file and the skull-stripped nifti file as inputs`

Reply:
https://github.com/neurodata/ndmg/pull/366/commits/65d670a04f209da4901ad6c0799cce039c700dcb